### DISCUSSION TITLE
I want add a language switch beside theme and notification ico on the right top bar menu, besides userMenu. HOW?

### DISCUSSION THREAD

**Discussion by guoxiangke** [laravel/nova-issues#6180] [Q&A]
<img width="725" alt="image" src="https://github.com/laravel/nova-issues/assets/1160703/2be0a429-322c-48e8-bdf8-d1469942767b">

Same here:
https://laracasts.com/discuss/channels/nova/add-menu-item-to-header

<img width="889" alt="image" src="https://github.com/laravel/nova-issues/assets/1160703/43ca7c09-3e46-4c2c-ae83-af009d0538d7">

**Comment by guoxiangke** [comment]
same https://github.com/laravel/nova-issues/discussions/6117

**Comment by forgecodex** [comment]
There are many aspects of the nova program that have to be compromised. For example, I want to add a clear cache to the userMenu. I hope that after clicking, it will prompt whether to clear the cache, similar to an action. However, after carefully looking at the content of the action, it is a circular model, so it will not work;

For example, I have many fields. I hope that there can be 2 or more columns in a row.

Neither can be achieved
